backend/runtimes/runtimes.go

```diff
@@ -1,18 +1,85 @@
 package runtimes
 
 import (
+	"go.autokitteh.dev/autokitteh/internal/backend/configset"
+	"go.autokitteh.dev/autokitteh/internal/backend/httpsvc"
 	"go.autokitteh.dev/autokitteh/internal/kittehs"
 	configruntimesvc "go.autokitteh.dev/autokitteh/runtimes/configrt/runtimesvc"
 	pythonruntime "go.autokitteh.dev/autokitteh/runtimes/pythonrt"
 	starlarkruntimesvc "go.autokitteh.dev/autokitteh/runtimes/starlarkrt/runtimesvc"
 	"go.autokitteh.dev/autokitteh/sdk/sdkruntimes"
 	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.uber.org/zap"
 )
 
-func New() sdkservices.Runtimes {
-	return kittehs.Must1(sdkruntimes.New([]*sdkruntimes.Runtime{
+type Config struct {
+	RemoteRunnerEndpoints []string `koanf:"remote_runner_endpoints"`
+	RemoteRunner          bool     `koanf:"enable_remote_runner"`
+	WorkerAddress         string   `koanf:"worker_address"`
+	// TODO: This is a hack to prevent running configure on pythonrt in each test
+	// which currently install venv everytime and takes a really long time
+	// need to find a way to share the venv once for all tests
+	PythonRTEnabled bool `koanf:"enable_python_runtime"`
+}
+
+var Configs = configset.Set[Config]{
+	Default: &Config{
+		RemoteRunnerEndpoints: []string{"localhost:9291"},
+		RemoteRunner:          false,
+		PythonRTEnabled:       true,
+	},
+	Test: &Config{
+		PythonRTEnabled: false,
+	},
+}
+
+func New(cfg *Config, l *zap.Logger) sdkservices.Runtimes {
+	runtimes := []*sdkruntimes.Runtime{
 		starlarkruntimesvc.Runtime,
 		configruntimesvc.Runtime,
-		pythonruntime.Runtime,
-	}))
+	}
+
+	//TODO: need to rethink
+	if cfg == nil {
+		cfg = Configs.Default
+	}
+
```

Comment:
Validate the `cfg.RemoteRunnerEnpoints` is valid (every one in `host:port` format, maybe even try to reach them).

Reply:
I'm not sure about host:port, it can be any valid ip or dns
but I do try to connect to each of them in the configure function and send health RPC and fail if it doesn't work